src/coreclr/src/System.Private.CoreLib/ILLinker/ILLink.Suppressions.cs

```diff
@@ -0,0 +1,276 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The existing folder name in mono is `ILLInk` not `ILLinker`. Can we name the folder consistently throughout the repo?